components/simple-paypal.jsx

```diff
@@ -177,7 +177,7 @@ var simplePaypal = React.createClass({
           <input type="hidden" name="cmd" value="_xclick-subscriptions"/>
           <input type="hidden" name="business" value="44ZHAVWJHTK2N"/>
           <input type="hidden" name="lc" value={this.props.paypalLocal}/>
-          <input type="hidden" name="item_name" value="Mozilla Foundation Recurring Donation"/>
+          <input type="hidden" name="item_name" value="Mozilla Foundation Monthly Donation"/>
```

Comment:
I thought we should be specific in the name about how often the donation occurs.